### ISSUE TITLE
A bunch of logos that I have created or converted

### ISSUE THREAD

**Issue by denilsonsa** [gilbarbara/logos#101]
**TL;DR:** I have a bunch of SVG logos that you may want to add to your repo.

Hi! I've been converting (sometimes creating from scratch) several SVG logos and saving them on my repository. Maybe you wanna look at them and incorporate them into your repository.

https://github.com/denilsonsa/denilsonsa.github.io/tree/master/icons

I always hand-edit the SVG code to make the code more readable and to remove inkscape-specific stuff. I leave a `<!-- comment -->` in each file pointing to the source material I used.

I've submitted some of them to https://github.com/johan/kilobyte-svg-challenge, and it would be great to also have them on your repository (which seems to be the best collection of SVG logos, AFAIK).

**Comment by gilbarbara** [comment]
Hey @denilsonsa 
Sweet, I already downloaded some and I'll add them soon.

However I edit all the logos manually and strip everything not used in the graphic itself and leave them with the same size and syntax. Is that ok?

[s]

**Comment by gilbarbara** [comment]
@denilsonsa And I already vectorized over 100 logos from raster images. Some of my friends call me crazy but I find it very soothing. :+1:

**Comment by gilbarbara** [comment]
Added some @ bb6c6084557a0c1da093a91b02d52471a1674660

**Comment by denilsonsa** [comment]
Well, it would be awesome to have a blog post or a screencast showing how you do it. Is it completely manual? Do you use any tracing tool (and later fine-tune the result)? And when the logo looks a bit complex? And when it is hard to find a high-res version?

By the way, I've added/updated a few logos today: https://github.com/denilsonsa/denilsonsa.github.io/commits/master/icons

> However I edit all the logos manually and strip everything not used in the graphic itself and leave them with the same size and syntax. Is that ok?

Of course it is okay! I see it is not too different from the code style from my SVGs (which can be summed up as: "keep it clean").

**Comment by gilbarbara** [comment]
@denilsonsa 
There isn't a clear process for vectorizing.
For simple geometric logos I prefer to draw the paths manually. For the rest I always try to trace it first in Illustrator while tweaking its options for path and curves accuracy but I rarely use the first result.
And always do it in a different layer of the original raster to use it as a guide.

But I also edit some of the original vector logos (svg/eps/pdf) cause I like to overlap the paths so there is no gap between them.
![overlap](https://cloud.githubusercontent.com/assets/31954/10120364/8d031d06-648d-11e5-9af9-71b9bcbc02c3.png)

And always convert the stroke to paths so it look the same ate every size.

[s]

**Comment by gilbarbara** [comment]
@denilsonsa continuing..
If it is too complex or I can't find a decent size version there's no magic. It will probably looks a bit different and most of the time doesn't worth the effort
I have a few logos that I would love to vectorize but I don't think I ever will..
